config/prefsConfig.go

```diff
@@ -13,4 +13,5 @@ type PreferencesConfig struct {
 	RedundancyLevel            int64
 	PreferredConnections       []string
 	FullArchive                bool
+	ConnectionWatcherType      string
```

Comment:
please leave `FullArchive bool` the last one due to memory alignment

Reply:
moved up one position